### PR TITLE
FIX: Removing unused code and fixing bug of generating csv in multiple periods graphs

### PR DESCRIPTION
**Pull request by hyahiaoui-ext**
## Description

While in multiple periods graphs we can't generate csv nor an image, this should take care of it also removing some unused code to clean up the code base.

**Fixes** # MON-14927

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/include/views/graphs/generateGraphs/generateImage.php [hunk 1]
```diff
@@ -95,8 +95,6 @@
     } else {
         die('Invalid token');
     }
-} else {
-    throw new \Exception('Username and token query strings must be set.');
```
Comment (TamazC) [inline www/include/views/graphs/generateGraphs/generateImage.php:99]: What's wrong with this condition ?

Comment (hyahiaoui-ext) [reply www/include/views/graphs/generateGraphs/generateImage.php:99]: @TamazC When you click on "generate image" it doesn't send token nor the username in the query strings so the generate image won't work, also i did add this "else" clause before when i sanitized queries in this file but i don't remember why exactly.

**Review by TamazC — APPROVED** [review]

File: www/include/views/graphs/graph-periods.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <div id="graphs" class="graph_content">
         <div class="graphZone">
           {foreach from=$periods item=period}
-          <div id="graph_wrapper_{$period.short}" data-graph-id="{$srv_id}" class="graph">
+          <div id="graph_wrapper_{$period.short}" data-graph-id="{$svc_id}" data-graph-period="{$period.short}" class="graph">
```
Comment (kduret) [inline www/include/views/graphs/graph-periods.html:14]: what is the purpose of `data-graph-period` attribute ?

**Review by jeremyjaouen — APPROVED** [review]

File: www/include/views/graphs/graph-periods.html (hunk 2 above)
Comment (hyahiaoui-ext) [reply www/include/views/graphs/graph-periods.html:14]: @kduret it holds the period of the graph (day, week, month) since they are seperated.

**Review by kduret — APPROVED** [review]

**Pull request merged**